### PR TITLE
presto chart

### PR DESCRIPTION
**Pull request by bivas**
Helm Chart for [PrestoDB](https://prestodb.io)

### CODE REVIEW

File: stable/presto/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{{- end -}}
+{{- end -}}
+
+{{- define "presto.worker" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
```
Comment (unguiculus) [inline stable/presto/templates/_helpers.tpl]: You need the suffix here as well.

File: stable/presto/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+{{- else -}}
+{{- printf "%s-%s-coordinator" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "presto.worker" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
```
Comment (unguiculus) [inline stable/presto/templates/_helpers.tpl]: You need the suffix here as well.

File: stable/presto/templates/_helpers.tpl [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "presto.coordinator" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
```
Comment (unguiculus) [inline stable/presto/templates/_helpers.tpl]: You need the suffix here as well.

File: stable/presto/templates/_helpers.tpl [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s-coordinator" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
```
Comment (unguiculus) [inline stable/presto/templates/_helpers.tpl]: You need the suffix here as well.

How about something like this:
```yaml
{{- define "presto.coordinator" -}}
{{ template "presto.fullname" . }}-coordinator
{{- end -}}
```

File: stable/presto/templates/configmap-worker.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
+{{- if gt .Values.server.workers 0.0 }}
```
Comment (unguiculus) [inline stable/presto/templates/configmap-worker.yaml]: You might want to try the `int` function. 

`{{- if gt (int .Values.server.workers) 0 }}`

File: stable/presto/templates/deployment-worker.yaml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/bash"]
+          args:
+            - {{ .Values.server.config.path }}/docker-presto.sh
+          volumeMounts:
+            - mountPath: {{ .Values.server.config.path }}
+              name: config-volume
+          livenessProbe:
+            exec:
+              command:
```
Comment (unguiculus) [inline stable/presto/templates/deployment-worker.yaml:42]: Use consistent list indentation. To match the prevailing style, indent one more level. Apply everywhere.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. Just a few things.

File: stable/presto/templates/_helpers.tpl (hunk 3 above)
Comment (bivas) [reply stable/presto/templates/_helpers.tpl]: What do you mean? This is the autogenerated version when running `helm create`

**Comment by bivas** [comment]
@unguiculus Updated with your comments

**Review by unguiculus — APPROVED** [review]

**Comment by purplexa** [comment]
This isn't actually a valid chart: the version (`0.6`) doesn't match SemVer, [despite Masterminds/semver acting like it does](https://github.com/Masterminds/semver/issues/75).

**Pull request merged**